docs/3-customizing-own-project.md

```diff
@@ -37,6 +37,90 @@ If you have built the sample project and have the pipeline working, it is very s
 
 1. Check in your changes and raise a PR to merge them into the master branch.
 
+## Enabling the F-measure testing capability
+
+The **LUIS F-measure testing** job measures performance of your LUIS model. This job performs the equivalent of the [LUIS batch testing](https://docs.microsoft.com/azure/cognitive-services/LUIS/luis-how-to-batch-test) that you can perform in the LUIS portal. It uses the [NLU.DevOps](https://github.com/microsoft/NLU.DevOps/blob/master/docs/Analyze.md) tool to execute the tests defined in the `luis-app/tests/verificationtests.json` test file against the newly built model and calculates the test results file, `statistics.json`, from which can be calculated the F1 score (or [F-measure](https://en.wikipedia.org/wiki/F-measure)). The results file from every run is saved to Azure blob storage and it contains:
+
+* For each Intent and Entity:
+  * Count of true positives for passing tests
+  * Count of true negatives for passing tests
+  * Count of false positives from failing tests
+  * Count of false negatives from failing tests
+  
+As shipped, the **LUIS F-measure testing** job is configured to run concurrently with the **Create LUIS Release** job and runs purely as an advisory and does not block the **LUIS CD** job from running should it fail. It is provided as an example of how to perform model quality testing within the workflow.
```

Comment:
Maybe "As shipped, the **LUIS F-measure testing** job is configured to run concurrently with the **Create LUIS Release** job and does not block the **LUIS CD** job from running should it fail. It runs purely as an advisory and is provided as an example of how to perform model quality testing within the workflow."

Reply:
That's better. Updated.